src/parser/jobs/pld/modules/Gauge.tsx

```diff
@@ -55,7 +55,6 @@ export default class Gauge extends CoreGauge {
 				A total of {this.oathGauge.overCap} gauge was lost due to exceeding the cap.
 			</Trans>,
 			severity: SEVERITY.MINOR,
-			value: this.oathGauge.overCap,
```

Comment:
This shouldn't be removed, and not part of a Buff Window change either.

Reply:
As above, this value had to be removed since Suggestion file was converted to TS and now takes a constructor object instead of an any.  The value property is part of TieredSuggestion but is not used by Suggestion.